components/form.js

```diff
@@ -1087,6 +1088,7 @@ export function Form ({
         await onSubmit(values, ...args)
       }
     } catch (err) {
+      if (err instanceof WalletPromptClosed) return
```

Comment:
I don't like how many layers away this is from where the error is thrown, but since `WalletPromptClosed` is semantically indeed more a submit error than success, it also kind of does make sense to throw it all the way up to here. :thinking: